Readme.md

```diff
@@ -0,0 +1,135 @@
+
+# First-Capstone
+
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Capstone-Project: Summer BootCamp 2023 ](#-capstone-project-summer-bootcamp-2023-)
+  - [🛠 Built With ](#-built-with-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Capstone-Project: Summer Music Festival <a name="about-project"></a>
+
+**[Capstone-Project: Summer Music Festival]** is about series of conscert prodution during Summer. the program will feature all different kind of music and artist .
+
+[Loom Video Presentation](https://www.loom.com/share/fec156053a8d4a5488170b52f51be1be)
+
+## 🛠 Built With <a name="built-with"></a>
+    HTML
+    CSS
+    Javascript
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Summer Music festival](https://github.com/Patfarmurs/Summer-Music-Festival/ )
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
[OPTIONAL] While you did an amazing job updating your read me file the link you added for your live demo directs the user to your repository. it would be nice if you deployed your project using github pages.